starport/templates/ibc/oracle.go

```diff
@@ -0,0 +1,370 @@
+package ibc
+
+import (
+	"embed"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/gobuffalo/genny"
+	"github.com/gobuffalo/plush"
+	"github.com/gobuffalo/plushgen"
+	"github.com/tendermint/starport/starport/pkg/multiformatname"
+	"github.com/tendermint/starport/starport/pkg/placeholder"
+	"github.com/tendermint/starport/starport/pkg/xgenny"
+	"github.com/tendermint/starport/starport/pkg/xstrings"
+)
+
+var (
+	//go:embed oracle/static/* oracle/static/**/*
+	fsOracleStatic embed.FS
+
+	// ibcOracleStaticTemplate is the template to scaffold a new static oracle templates in an IBC module
+	ibcOracleStaticTemplate = xgenny.NewEmbedWalker(fsOracleStatic, "oracle/static/")
+
+	//go:embed oracle/dynamic/* oracle/dynamic/**/*
+	fsOracleDynamic embed.FS
+
+	// ibcOracleDynamicTemplate is the template to scaffold a new dynamic oracle templates in an IBC module
+	ibcOracleDynamicTemplate = xgenny.NewEmbedWalker(fsOracleDynamic, "oracle/dynamic/")
+)
+
+// OracleOptions are options to scaffold an oracle in a IBC module
```

Comment:
```suggestion
// OracleOptions are options to scaffold an oracle query in a IBC module
```

For `OracleOptions` this id ok to me to call it like this because this is the component related to oracles. But when we explain what we actually scaffold in the chain, we should always mention oracle queries